buckets/archive/tracker.go

```diff
@@ -88,7 +89,20 @@ func (t *Tracker) run() {
 
 						ctx, cancel := context.WithTimeout(t.ctx, time.Second*5)
 						defer cancel()
-						reschedule, cause, err := t.trackArchiveProgress(ctx, a.BucketKey, a.DbID, a.DbToken, a.JID, a.BucketRoot)
+
+						var ffsInfo *mdb.FFSInfo
+						if account, err := t.colls.Accounts.Get(ctx, a.Owner); err == nil {
+							ffsInfo = account.FFSInfo
+						} else if user, err := t.colls.Users.Get(ctx, a.Owner); err == nil {
+							ffsInfo = user.FFSInfo
+						}
```

Comment:
Here we are in the tracker where we pull out the newly persisted `Owner` so we can look up the `Account` or `User` to get their FFS info